extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -128,15 +116,17 @@ export class AmpConsent extends AMP.BaseElement {
           new ConsentUI(this, dict({}), config.getPostPromptUI());
     }
 
-    this.consentConfig_ = config.getConsentConfig();
+    const consentConfigDepr = config.getConsentConfig();
 
     // ConsentConfig has verified that there's one and only one consent instance
-    this.consentInstanceId_ =
-        Object.keys(/** @type {!Object} */ (this.consentConfig_))[0];
+    this.consentId_ =
+        Object.keys(/** @type {!Object} */ (consentConfigDepr))[0];
```

Comment:
Are we certain Object.keys will always sort the the value to be the first? Why not just use the key using consentConfigDepr['key-that-I-want']? 😄